internal/util/wrappers/qn_wrapper.go

```diff
@@ -148,6 +148,10 @@ func (qn *qnServerWrapper) Delete(ctx context.Context, in *querypb.DeleteRequest
 	return qn.QueryNode.Delete(ctx, in)
 }
 
+func (qn *qnServerWrapper) CheckHealth(ctx context.Context, req *milvuspb.CheckHealthRequest, opts ...grpc.CallOption) (*milvuspb.CheckHealthResponse, error) {
+	return qn.QueryNode.CheckHealth(ctx, req)
```

Comment:
`CheckHealth` is useless here, `qn_wrapper` is used for delegator to call rpc to worker on same query node

Reply:
Yes, it only implements the gRPC interface of the QueryNode client for QnServerWrapper.